commands/shooter/resetshooting.py

```diff
@@ -0,0 +1,27 @@
+from commands2 import CommandBase
+from wpimath.geometry import Rotation2d
+
+from subsystems.shootingsubsystem import ShootingSubsystem
+
+
+class ResetShooting(CommandBase):
+    def __init__(self, shooting: ShootingSubsystem) -> None:
+        CommandBase.__init__(self)
+        self.setName(__class__.__name__)
+        self.shooting = shooting
+        self.addRequirements([shooting])
+
+    def initialize(self) -> None:
+        print(f"Command: {self.getName()}")
+
+    def execute(self) -> None:
+        self.shooting.setAsStartingPosition()
+        self.shooting.rotateTurret(Rotation2d.fromDegrees(0))
```

Comment:
Should this be 180 (aka `kTurretCenterAngle`)?

Reply:
rotates the turret to turret forward - I attempted to use 180 as forward but there was generally so many issues caused, 0 forward is more consistent